binderhub/static/js/index.js

```diff
@@ -144,7 +143,7 @@ function build(providerSpec, log, path, pathType) {
   image.onStateChange('*', function(oldState, newState, data) {
     if (data.message !== undefined) {
       log.write(data.message);
-      log.fit();
+      fit(log); //log.fit();
```

Comment:
Turning this around so that you don't need `applyAddon` is what jupyter lab does as well and seems to work.